.github/workflows/acceptance.yml

```diff
@@ -1,7 +1,6 @@
 name: acceptance
 on:
   push:
-    branches: [ main ]
```

Comment:
8 commits in total, plus commits that shouldn't be here at all, this requires redesign

Reply:
done